mypyc/emitmodule.py

```diff
@@ -67,6 +68,33 @@
 # A list of (file name, file contents) pairs.
 FileContents = List[Tuple[str, str]]
 
+# The C standard specifies that any number of valid hexadecimal characters are
```

Comment:
I think it's worth moving this out of `emitmodule` at this point. `mypyc.cstring`, maybe?

Reply:
(And then encode_as_c_string and encode_bytes_as_c_string too)